mirror-service/src/main/java/org/acme/client/NvdClient.java

```diff
@@ -0,0 +1,91 @@
+package org.acme.client;
+
+import io.github.jeremylong.nvdlib.NvdCveApi;
+import io.github.jeremylong.nvdlib.NvdCveApiBuilder;
+import io.github.jeremylong.nvdlib.nvd.DefCveItem;
+import org.apache.kafka.streams.processor.api.ContextualProcessor;
+import org.apache.kafka.streams.processor.api.ProcessorContext;
+import org.apache.kafka.streams.processor.api.Record;
+import org.apache.kafka.streams.state.KeyValueStore;
+import org.apache.kafka.streams.state.StoreBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.WebApplicationException;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Client for the NVD REST API.
+ */
+public class NvdClient extends ContextualProcessor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NvdClient.class);
```

Comment:
Wev are not using logger in this class. Should we just remove it to clean it up ?